src/wizard/javascript/angular.md

```diff
@@ -23,11 +23,19 @@ You should `init` the Sentry browser SDK as soon as possible during your applica
 import { enableProdMode } from '@angular/core';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import * as Sentry from '@sentry/angular';
+import { Integrations } from "@sentry/tracing";
 
 import { AppModule } from './app/app.module';
 
 Sentry.init({
   dsn: '___PUBLIC_DSN___',
+  integrations: [
+    new Integrations.BrowserTracing({
+      tracingOrigins: ["localhost", "https://yourserver.io/api"],
+      routingInstrumentation: Sentry.routingInstrumentation,
+    }),
+  ],
+  tracesSampleRate: 1.0,
```

Comment:
We need to add the message about traces sample rate like we do for react/python wizard

Reply:
good catch
added